playbooks/roles/cluster/tasks/main.yml

```diff
@@ -1,2 +1,33 @@
 ---
+- hosts: localhost
+  connection: local
+  tasks:
+    - name: Copy the tower_github_scm key to server.
+      copy:
+        content: '{{ tower_github_scm }}'
+        dest: '/home/{{ credential_git_user }}/.ssh/tower_github_scm'
+        mode: 0600
+        owner: '{{ cluster_credential_git_user }}'
+        group: '{{ cluster_credential_git_user_group }}'
+    - name: Clone the credential repo
```

Comment:
Similar to my previous point, it might be worth moving the cloning of the credential repo to a more generic location as we'll need to perform this for cluster, integreatly and tower bootstrapping tasks